utilities/deep_inspection.py

```
@@ -319,45 +323,113 @@ def wait_for_conversion_complete(conversion: Conversion, timeout: int) -> None:
+    """Validate inspectionResult data from a DI Conversion CR.
+
+    Args:
+        results (dict[str, Any] | None): The inspectionResult dict.
+        vm_name (str): VM name for error context.
+        context (str): Additional context (e.g., conversion name or plan name).
+
+    Raises:
+        ValueError: If results are missing or incomplete.
+    """
+    if not results:
+        raise ValueError(f"{context}, VM '{vm_name}': inspectionResult is empty")
```

Ruff (0.16.0)

[warning] Avoid specifying long messages outside the exception class

(TRY003)